src/pages/Registration/RegistrationContainer.jsx

```diff
@@ -0,0 +1,76 @@
+import Registration from "./Registration";
+import { useState, useEffect } from "react";
+import formatRequest from "../../helpers/formatRequest";
+import { Redirect } from "react-router";
+
+const RegistrationContainer = () => {
+  const [error, setError] = useState([]);
+  const [userCreated, setUserCreated] = useState(false);
+  const [citiesArray, setCitiesArray] = useState([]);
+
+  const getCitiesList = async () => {
+    try {
+      const response = await formatRequest("/city", "GET");
```

Comment:
it's better to store cities in a redux store and create actions for that. because they're going to be cached and used everywhere. 

it's better to check if there  are some cities placed in redux, so there is no need in sending the request to back end again when page's open

Reply:
Done.